Packs/Base/Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1

```diff
@@ -268,7 +268,14 @@ class DemistoObject {
         if ( -not $this.IsIntegration ) {
             throw "Method not supported"
         }
-        return $this.ServerRequest(@{type = "executeCommand"; command = "getIntegrationContext"; args = @{ } }).context
+        $integration_context = $this.ServerRequest(@{type = "executeCommand"; command = "getIntegrationContext"; args = @{ } })
+        # When Demisto Version is greater equal then "6.0.0".  integration_context will be under "context" attribute.
+        $this.DemistoVersion().version -match "\d{1,2}\.\d{1,2}\.\d{1,2}" | Out-Null
```

Comment:
Not sure why we are doing this. This logic should be merged into the VersionEqualGreaterThen.

Reply:
Just for unit-test reason, you right I will mock it and insert it in to VersionEqualGreaterThen